### DISCUSSION TITLE
Support for Alpine 3.18

### DISCUSSION THREAD

**Discussion by userid0x0** [crazy-max/docker-alpine-s6#64] [General]
Hi,
I really appreciate your work, good job. Is there a schedule to support Alpine 3.18?
I tested Alpine 3.18 locally using one of my images without any issues so far.
Regards,
Alex

**Comment by crazy-max** [comment]
Just merged https://github.com/crazy-max/docker-alpine-s6/pull/65, will do a release shortly

**Comment by userid0x0** [comment]
Yet another question. I see there is now a `3.18-edge` tag but no `3.18-x.x.x.x` is that intended or is it WIP?

**Comment by crazy-max** [comment]
Release builds on going: https://github.com/crazy-max/docker-alpine-s6/actions
